hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/utils/FrameUtils.java

```diff
@@ -370,4 +370,13 @@ private static boolean isQualifiedDelegate(@NonNull final Address recipient, @No
         return isLongZero(recipient)
                 && contractsConfigOf(frame).permittedDelegateCallers().contains(numberOfLongZero(recipient));
     }
+
+    public static boolean isPrecompileEnabled(
+            @NonNull final Address precompileAddress, @NonNull final MessageFrame frame) {
```

Comment:
Can this be improved to list traverse only once at config load.
Then future calls here would only check a map?